feedback.md

```diff
@@ -0,0 +1,37 @@
+AMBULO
+======
+<br />
+
+| Team Member | Role                 |
+| ------------| -------------------- |
+| Anton Zheng | Dev Lead             |
+| Jenny Yang  | Repo Master          |
+| Karen Lee   | User Experience Lead |
+| Thoa Nguyen | Test Lead/PM         |
+
+<br />
+
+## DESCRIPTION
+A hiking web-based application that uses Trail API for locations and Instagram API for images of trails. 
+Users will be able to view and filter trails on a map. When a user clicks on a trail marker, 
+they can get instant information such as weather conditions. 
```

Comment:
Is the weather condition included in the trail API? Would this require another API?